scripts/scil_fibertube_tracking.py

```diff
@@ -122,6 +127,35 @@ def _build_arg_parser():
              '(based on the propagator\'s definition of invalid) \n'
              'are never added.')
 
+    ftod_g = p.add_argument_group(
+        'ftODF Options',
+        'Options required if you want to perform fibertube tracking using\n'
+        'fibertube orientation distribution (ftODF).\n'
+        'If you\'re not familiar with these options, please ignore them.')
```

Comment:
Is this necessary? Maybe mention to go read the demo for more information about the ftODF